include/tvm/tir/transform.h

```diff
@@ -492,6 +492,39 @@ TVM_DLL Pass ConvertForLoopsToSerial();
  */
 TVM_DLL Pass UnifiedStaticMemoryPlanner();
 
+/*!
```

Comment:
Thanks for the great comment @vinx13, is it possible (maybe in a future PR) to walk through an example of the process to better understand what it looks like? can be "hello world" style but I think would help illustrate how it works. Maybe the test cases are the best example, if so maybe just update this comment to point to the test case?

Reply:
Thanks for the suggestion, I have update the comment to include an example